src/SST.jl

```diff
@@ -0,0 +1,28 @@
+struct SST{S}
```

Comment:
Docstring please
Also comments saying what the fields main

Reply:
We should just all it the `StanfordSentimentTreebank` 
SST it not a clear acronym

Reply:
Please tell me the docstring I added is fine or should I do something else.